src/ansys_sphinx_theme/__init__.py

```diff
@@ -110,9 +110,56 @@ def get_autoapi_templates_dir_relative_path(path: pathlib.Path) -> str:
         A string rerpesenting the relative path to the autoapi templates.
 
     """
-    return os.path.relpath(
-        str(AUTOAPI_TEMPLATES_PATH.absolute()), start=str(path.parent.absolute())
+    return os.path.relpath(str(AUTOAPI_TEMPLATES_PATH.absolute()), start=str(path.absolute()))
+
+
+def add_autoapi_theme_option(app: Sphinx) -> None:
+    """Add the autoapi template path to the theme options.
+
+    Parameters
+    ----------
+    app : ~sphinx.application.Sphinx
+        Application instance for rendering the documentation.
+    """
+    autoapi = app.config.html_theme_options.get("autoapi", {})
+    if not autoapi:
+        return
+    required_extensions = ["sphinx_jinja", "sphinx_design"]
+
+    for extension in required_extensions:
+        if extension not in app.config["extensions"]:
+            app.config["extensions"].append(extension)
+    AUTOAPI_OPTIONS = [
+        "members",
+        "undoc-members",
+        "show-inheritance",
+        "show-module-summary",
+        "special-members",
+    ]
```

Comment:
Is there a way of customizing these? Not all projects are using the `special-members`, for instance.

Reply:
yes , they can give these options as a list in theme options.